### ISSUE TITLE
Redo Documentation

### ISSUE THREAD

**Issue by teknogeek** [nmpiazza/Hermes#12]

**Comment by nmpiazza** [comment]
We should consider setting up [Sphinx](http://www.sphinx-doc.org/en/stable/) for documentation. It's pretty standard and will parse the google comment style listed above.

For setup and other documentation we should probably prioritize the GitHub wiki.

**Comment by teknogeek** [comment]
Can Sphinx read the google documentation formatting originally linked?

**Comment by nmpiazza** [comment]
Yes, it supports that. I've used this on other projects recently.

**Comment by nmpiazza** [comment]
Example documentation that's a little easier to read than their outline can be found here
https://github.com/nmpiazza/Hermes/blob/76c9c0e3efb9eef928b749a20a3806e1131ba738/app/mod_check/HTTP.py#L8-L24

**Comment by teknogeek** [comment]
Thanks for that example, I like the look of that a lot
